packages/client/utils/relay/setTemplateId.ts

```diff
@@ -0,0 +1,19 @@
+import {commitLocalUpdate} from 'react-relay'
+import {ITeam, MeetingTypeEnum} from '../../types/graphql'
+import Atmosphere from '../../Atmosphere'
+
+const setTemplateId = (atmosphere: Atmosphere, teamId: string, templateId: string | null, meetingType: MeetingTypeEnum) => {
```

Comment:
nit: lets rename this to show it's changing the active template.
maybe `activateTemplate` or `setActiveTemplate`?